src/js/Mixins/Snapping.js

```diff
@@ -212,8 +212,11 @@ const SnapMixin = {
         layer instanceof L.Polyline ||
         layer instanceof L.Marker ||
         layer instanceof L.CircleMarker
+
       ) {
-        layers.push(layer);
+          if(layer.options.snapIgnore !== true){
```

Comment:
Could this condition be part of the above if-clause in line 211?

Reply:
Fixed